packages/color-area/src/ColorArea.ts

```diff
@@ -113,48 +103,48 @@ export class ColorArea extends SpectrumElement {
 
     @property({ type: Number })
     public get x(): number {
-        return this._x;
+        return this.colorController.color.hsv.s / 100;
     }
 
     public set x(x: number) {
         if (x === this.x) {
             return;
         }
         const oldValue = this.x;
-        this._x = x;
         if (this.inputX) {
             // Use the native `input[type='range']` control to validate this value after `firstUpdate`
             this.inputX.value = x.toString();
-            this._x = this.inputX.valueAsNumber;
+            this.colorController.color.set(
+                's',
+                this.inputX.valueAsNumber * 100
+            );
+        } else {
+            this.colorController.color.set('s', x * 100);
```

Comment:
Yes, we are multiplying x and y coordinates by 100 to convert coordinate values into saturation and value respectively. We are doing this 4 times in the code, Twice for x coordinate and twice for y coordinates.
Thus it can't be overriden out of the box.

Reply:
Again this setX is very specific to color area and doesn't exist in other color components.